app/main.py

```diff
@@ -11,8 +11,7 @@ def __repr__(self):
     def __add__(self, other):
         if isinstance(other, type(self)):
```

Comment:
explicitly show the type, now type(self) looks very confusing, specify the class like - isinstance(other, Distance)
change it in all ur methods ;D

Reply:
and fix flake8